scalingo/utils_slice.go

```diff
@@ -1,5 +1,18 @@
 package scalingo
 
+// Filter filters slice by using the test function.
+// It returns a filtered slice.
```

Comment:
I don't really like the comment. It uses the word "filter" everywhere without explaining what it means. What happens if the test function returns `true`? Does it keep the element? Reject it?